apollo-router/src/services/layers/persisted_queries/mod.rs

```diff
@@ -985,11 +983,14 @@ mod tests {
     }
 
     #[tokio::test(flavor = "multi_thread")]
+    #[cfg(not(all(not(feature = "ci"), target_os = "macos")))]
     async fn pq_layer_freeform_graphql_with_safelist_log_unknown_true() {
```

Comment:
Maybe we can add it to the list of flaky tests, rather than entirely disabling it?

This doesn't fail all the time. When we looked into it last, it would always pass if you run *only* this test with `cargo t pq_layer_freeform_graphql_with_safelist_log_unknown_true`. But we did not find the root cause despite looking at it pretty deeply.